src/backend/src/prisma-query-args/description-bullets.query-args.ts

```diff
@@ -0,0 +1,5 @@
+import { Prisma } from '@prisma/client';
+
+export const descBulletArgs = Prisma.validator<Prisma.Description_BulletArgs>()({
```

Comment:
should be exported as the default like in risks

and lets rename it to `descriptionBulletQueryArgs`